samples/common/src/main/java/com/bumble/appyx/samples/common/profile/Profile.kt

```diff
@@ -0,0 +1,96 @@
+package com.bumble.appyx.samples.common.profile
+
+import android.os.Parcelable
+import androidx.annotation.DrawableRes
+import com.bumble.appyx.samples.common.R
+import kotlinx.parcelize.Parcelize
+import kotlin.random.Random
+
+@Parcelize
+@Suppress("MagicNumber")
+data class Profile(
+    val name: String,
+    val age: Int = Random.nextInt(20, 35),
+    val city: String = "London",
+    val uni: String = "Oxford University",
+    @DrawableRes val drawable: Int = allProfiles.random().drawable
```

Comment:
rename to `drawableRes` or `drawableId`? Looking elsewhere in the code I thought this data class was keeping the actual image in memory.